tools/gbench/report.py

```diff
@@ -78,30 +84,82 @@ def filter_benchmark(json_orig, family, replacement=""):
     for be in json_orig['benchmarks']:
         if not regex.search(be['name']):
             continue
-        filteredbench = copy.deepcopy(be) # Do NOT modify the old name!
+        filteredbench = copy.deepcopy(be)  # Do NOT modify the old name!
         filteredbench['name'] = regex.sub(replacement, filteredbench['name'])
         filtered['benchmarks'].append(filteredbench)
     return filtered
 
 
-def generate_difference_report(json1, json2, use_color=True):
+def generate_difference_report(
+        json1,
+        json2,
+        utest=False,
+        utest_alpha=0.05,
+        use_color=True):
+    assert utest is True or utest is False
```

Comment:
this should be under the docstring. also, if you care about the type being boolean specifically, maybe add type annotations :)

otherwise, it's up to the caller to be truthy or not. this is Python, after all.

Reply:
True.
I'm just following pre-existing practice e.g. with `color_format()` doing this.